sjb/config/test_cases/test_branch_openshift_ansible_logging.yml

```diff
@@ -85,107 +63,141 @@ extensions:
+        sudo firewall-cmd --permanent --zone dockerc --add-port 53/udp
+        sudo firewall-cmd --permanent --zone dockerc --add-port 8053/udp
+        sudo firewall-cmd --reload
+        # this is specific to AWS - other platforms will need to get public ip and hostname elsewhere
+        metadata_endpoint="http://169.254.169.254/latest/meta-data"
+        public_hostname="$( curl -s "${metadata_endpoint}/public-hostname" )"
+        public_ip="$( curl -s "${metadata_endpoint}/public-ipv4" )"
+        sudo oc cluster up --public-hostname="${public_hostname}" --routing-suffix="${public_ip}.xip.io"
+        # change the config
+        # allow externalIPs, and kibana UI access
+        SERVER_CONFIG_DIR=/var/lib/origin/openshift.local.config
+        # add loggingPublicURL so the OpenShift UI Console will include a link for Kibana
+        # this part stolen from util.sh configure_os_server()
+        sudo cp ${SERVER_CONFIG_DIR}/master/master-config.yaml ${SERVER_CONFIG_DIR}/master/master-config.orig.yaml
+        ocver=$( oc version | awk -F'[ v.-]+' '/^oc / {print $2 $3 $4}' )
+        patchcmd="sudo oc ex config patch"
+        if [ $ocver -lt 380 ] ; then
+            patchcmd="docker exec origin openshift ex config patch"
+        fi
+        $patchcmd ${SERVER_CONFIG_DIR}/master/master-config.orig.yaml \
+            --patch="{\"assetConfig\": {\"loggingPublicURL\": \"https://kibana.${public_ip}.xip.io\"}}" | \
+            sudo tee ${SERVER_CONFIG_DIR}/master/master-config.yaml > /dev/null
+        sudo cp ${SERVER_CONFIG_DIR}/master/master-config.yaml ${SERVER_CONFIG_DIR}/master/master-config.save.yaml
+        $patchcmd ${SERVER_CONFIG_DIR}/master/master-config.save.yaml \
+            --patch="{\"networkConfig\": {\"externalIPNetworkCIDRs\": [\"0.0.0.0/0\"]}}" | \
+            sudo tee ${SERVER_CONFIG_DIR}/master/master-config.yaml > /dev/null
+        # restart cluster so changes will take effect
+        sudo oc cluster down
+        # have to restart docker while oc cluster is down, after it has been initialized, or inter-pod
+        # networking does not work
+        sudo systemctl restart docker
+        sudo oc cluster up --use-existing-config --public-hostname="${public_hostname}" --routing-suffix="${public_ip}.xip.io"
     - type: "script"
       title: "expose the kubeconfig"
       script: |-
+        if [ ! -d $HOME/.kube ] ; then
+          mkdir $HOME/.kube
+        fi
+        sudo cp /var/lib/origin/openshift.local.config/master/admin.kubeconfig $HOME/.kube/config
+        sudo chown $USER $HOME/.kube/config
+        sudo mkdir -p /etc/origin/master
+        sudo cp /var/lib/origin/openshift.local.config/master/admin.kubeconfig /etc/origin/master/admin.kubeconfig
         sudo chmod a+x /etc/ /etc/origin/ /etc/origin/master/
         sudo chmod a+rw /etc/origin/master/admin.kubeconfig
-    - type: "script"
-      title: "ensure built version of origin is installed"
```

Comment:
If you're installing `origin-clients` you still 100% need this step to ensure you're installing the correct version. It's runtime is <1s but the peace of mind is huge.

Reply:
not sure what you mean

Reply:
Don't remove this step